dadi/lib/index.js

```diff
@@ -946,34 +942,36 @@ function onListening (e) {
   var engines = Object.keys(templateStore.getEngines())
   var enginesInfo = engines.length ? engines.join(', ') : 'None found'.red
 
-  // check that our API connection is valid
-  help.isApiAvailable((err, result) => {
-    if (err) {
-      dadiBoot.error(err)
-      process.exit(0)
-    } else if (config.get('env') !== 'test') {
-      dadiBoot.started({
-        server: `${config.get('server.protocol')}://${config.get(
-          'server.host'
-        )}:${config.get('server.port')}`,
-        header: {
-          app: config.get('app.name')
-        },
-        body: {
-          Protocol: config.get('server.protocol'),
-          Version: version,
-          'Node.js': nodeVersion,
-          Engine: enginesInfo,
-          Environment: config.get('env')
-        },
-        footer: {
-          'DADI API': config.get('api.enabled')
-            ? `${config.get('api.host')}:${config.get('api.port')}`
-            : '\u001b[31mNot enabled\u001b[39m'
-        }
-      })
+  if (config.get('env') !== 'test') {
+    let footer = {}
+
+    if (config.get('api').host) {
+      let apiKey = config.get('api').type ? config.get('api') : 'DADI API'
+      footer[apiKey] = config.get('api').host
+    } else {
+      for (let api in config.get('api')) {
+        footer[api === 'dadiapi' ? 'DADI API' : api] =
```

Comment:
Minor detail, but personally not a big fan of computing that expression inside the array index, I think it makes it a bit hard to read. I would do the following, which would also make it consistent with lines 949-950:

```js
      for (let api in config.get('api')) {
        let key = api === 'dadiapi' ? 'DADI API' : api

        footer[key] =
          config.get('api')[api].host || config.get('api')[api].type
      }
```